### PR TITLE
Workshop survey cron job

### PR DESCRIPTION
**Pull request by mehalshah**
Extend the existing cron job to publish results for other workshops

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -51,6 +51,7 @@ npm-debug.log
 /globals.yml
 /levelbuilder_ci_last_run
 /locals.yml
+/log
```
Comment (aoby) [inline .gitignore]: For cleaner history / blame, please pull this out into its own PR. You can assign me as a reviewer and merge immediately. 👍

File: bin/cron/calculate_workshop_survey_results [hunk 2]
```diff
@@ -15,6 +17,24 @@ def main
     course_scores[course] = get_score_for_workshops(workshops)
   end
 
+  course_scores['CSP Local Summer Workshops'] = summarize_workshop_surveys(
+    Pd::Workshop.in_state(Pd::Workshop::STATE_ENDED).where(course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP),
+    nil,
+    false
```
Comment (aoby) [inline bin/cron/calculate_workshop_survey_results:23]: Should these be named parameters on `summarize_workshop_surveys` so it's more clear? Then also you would only need to specify those differ from the defaults.

File: dashboard/app/helpers/pd/workshop_survey_results_helper.rb [hunk 3]
```diff
@@ -12,17 +12,20 @@ module Pd::WorkshopSurveyResultsHelper
   #
   # @param surveys List of TeacherconSurveys or LocalWorkshopSurveys
   # @param facilitator Facilitator name to restrict responses for
+  # @param facilitator_breakdown Whether to have a facilitator breakdown
   # @returns Hash representing an average of all the respones, or array of free text responses
-  def summarize_workshop_surveys(surveys, facilitator = nil)
+  def summarize_workshop_surveys(workshops, facilitator = nil, facilitator_breakdown = true)
```
Comment (aoby) [inline dashboard/app/helpers/pd/workshop_survey_results_helper.rb]: Suggestion: turn the optional parameters into keyword arguments (`facilitator: nil, facilitator_breakdown: true`), so they can be specified more clearly and only as needed.

File: dashboard/app/helpers/pd/workshop_survey_results_helper.rb [hunk 4]
```diff
@@ -12,17 +12,20 @@ module Pd::WorkshopSurveyResultsHelper
...
+    surveys = workshops.flat_map(&:survey_responses)
+
     raise 'Currently just summarizes Local Summer and Teachercon surveys' unless
       surveys.all? {|survey| survey.is_a? Pd::TeacherconSurvey} ||
         surveys.all? {|survey| survey.is_a? Pd::LocalSummerWorkshopSurvey}
 
     return Hash.new if surveys.empty?
 
-    options = surveys.first.class.options
+    questions = surveys.first.class.options
```
Comment (aoby) [inline dashboard/app/helpers/pd/workshop_survey_results_helper.rb:28]: questions = options?

**Comment by mehalshah** [comment]
I think a merge has gone wrong here. Fixing
